Toolchain/Patches/gcc.patch

```diff
@@ -129,10 +129,10 @@ diff -Naur gcc-11.1.0-RC-20210420/gcc/config/serenity.h gcc-11.1.0-RC-20210420.s
 +#define LINK_SPEC "%{shared:-shared} %{static:-static} %{!static: %{rdynamic:-export-dynamic} %{!fbuilding-libgcc:%{!nodefaultlibs:-lgcc_s} -dynamic-linker /usr/lib/Loader.so}}"
 +
 +#undef CC1_SPEC
-+#define CC1_SPEC "-fno-exceptions -ftls-model=initial-exec %{!fno-pic:%{!fno-PIC:%{!fpic:%{!fPIC: -fPIC}}}} -fno-semantic-interposition"
++#define CC1_SPEC "-ftls-model=initial-exec %{!fno-pic:%{!fno-PIC:%{!fpic:%{!fPIC: -fPIC}}}} -fno-semantic-interposition"
 +
 +#undef CC1PLUS_SPEC
-+#define CC1PLUS_SPEC "-fno-exceptions -ftls-model=initial-exec"
```

Comment:
:catdogalert:

Reply:
That's intentional, actually. Previously we'd patch that out for the userland toolchain with sed. :)

Reply:
Oh.